howto/unstructured_dataset.md

```diff
@@ -0,0 +1,18 @@
+# Finetuning on an unstructured dataset
+
+While most scripts were made to finetune on instruction datasets, it is possible to finetune on any dataset, this is useful for experimentation while not being as expensive as training a full model.
+
+This guide is only to prepare the finetuning, as either LoRA or Adapter-v1 methods support this dataset type!
+
+## Preparation
+
+1. Gather your text into an input file named `input.txt`
+2. Prepare it into train and validation sets with the follwoing script.
```

Comment:
Small typo: follwoing -> following.

It's a matter of preference, but instead of

> Prepare it into train and validation sets with the follwoing script.

you could also say

> Divide the the data into training and validation sets using the following script: